components/kyma-environment-broker/internal/orchestration/dto.go

```diff
@@ -0,0 +1,12 @@
+package orchestration
+
+import "github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+
+type UpgradeOrchestrationDTO struct {
```

Comment:
According to documentation, this kind of DTO would be used both in response on `GET /orchestrations/{orchestration_id}` and when calling `POST /upgrade/kyma` - and it contains `Parameters` of a `Orchestration` - consider renaming it to just `Parameters`. Using it from package will look like this:
```golang
dto := orchestration.Parameters{}
```